app/assets/javascripts/student_profile/record_service.js

```diff
@@ -78,7 +78,8 @@
     },
 
     onProvidedByEducatorTyping: function(event) {
-      this.setState({ providedByEducatorName: event.target.value });
+      var updatedName = (event.target.value != "") ? event.target.value : null;
```

Comment:
(Cont. from above). I'd say use an empty string to mean empty, instead of using `null` to mean empty.